examples/user_guide/11-Transforming_Elements.ipynb

```diff
@@ -65,7 +65,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "expr = np.sin(hv.dim('x')*10+5)\n",
+    "expr = np.sin(hv.dim('x').df*10+5)\n",
```

Comment:
Is this .df intentional?  It does not seem to be needed here...

Reply:
Damn, I had thought you had caught that in your suggestions so didn't look further.

Reply:
And I thought _you_ were changing that one.  :-) No big deal, just checking...

Reply:
I pushed a fix to master.